### DISCUSSION TITLE
Additional _status?

### DISCUSSION THREAD

**Discussion by hai-qiu-bpc** [payloadcms/payload#3146] [Q&A]
Is there any way to add an additional status other than (draft, published) right now.
E.g one between draft and published?

We are deploying on a test and production server right now. And we are thinking of eliminating that but currently with only those 2 statuses it's not possible. Another one would be great to fully deploy the change in the content to production somehow.
Anybody has an idea??

**Comment by denolfe** [comment]
What are you trying to achieve with the additional status? Some sort of "In Review" status?

You could likely add another field to your document that can be updated as needed via a hook. If you give some more info, I can direct you accordingly.

**Reply by hai-qiu-bpc** [reply]
Hi, yes kind of. I want to use draft for, e.g. start writing content but want to finish them later. 
In review. Want to display the content to view it on my frontend as a test.
Published. Published to the actual site for my clients.

I hope it's clear. Basically the new status 'in review' should have the same properties as draft or published that I can request content via the api with asking for status = in review

how can you recommend to do that via a hook?

**Reply by denolfe** [reply]
Have a look at this example, as it looks to solve a lot of what you're desiring: https://github.com/payloadcms/payload/tree/master/examples/draft-preview/payload
